src/include/common/mask.h

```diff
@@ -76,6 +80,9 @@ class Roaring64BitmapSemiMask : public RoaringBitmapSemiMask {
           roaring(std::make_shared<roaring::Roaring64Map>()) {}
 
     void mask(common::offset_t nodeOffset) override { roaring->add(nodeOffset); }
+    void maskRange(common::offset_t startNodeOffset, common::offset_t endNodeOffset) override {
```

Comment:
The 32-bit semi-mask functions are covered but not the 64-bit ones. It probably isn't realistic to have scans that go past 32-bit range in our CI so that's probably why.